internal/pkg/controller/controller.go

```diff
@@ -0,0 +1,198 @@
+package controller
+
+import (
+	"time"
+	"fmt"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	informerruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/util/wait"
+	errorHandler "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	updateOnChangeAnnotation = "reloader.stakater.com.io/update-on-change"
+	// AllNamespaces as our controller will be looking for events in all namespaces
+	AllNamespaces = "temp-reloader"
+)
+
+// Event indicate the informerEvent
+type Event struct {
+	key          string
+	eventType    string
+	namespace    string
```

Comment:
`namespace` is unused

Reply:
We won't be needing the namespace in the event. This seems to be a direct copy from chowkidar. Please don't copy paste blindly. Try to understand how reloader should work and write code based on that yourself.